INFO:root:Contract /dataset/bad_randomness/lottery.sol:Lottery:
INFO:symExec:Running, please wait...
Traceback (most recent call last):
  File "/root/osiris/intFlow.py", line 349, in multiplication_overflow_check
    if s.check() == sat:
  File "/root/osiris/z3/z3.py", line 6246, in check
    r = Z3_solver_check_assumptions(self.ctx.ref(), self.solver, num, _assumptions)
  File "/root/osiris/z3/z3core.py", line 3401, in Z3_solver_check_assumptions
    r = _elems.f(a0, a1, a2, a3)
  File "/root/osiris/symExec.py", line 2868, in handler
    raise Exception("timeout")
Exception: timeout
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 53.3%
INFO:symExec:	  Arithmetic bugs: 	 True
INFO:symExec:	  └> Overflow bugs: 	 True
/dataset/bad_randomness/lottery.sol:Lottery:11:2
contract Lottery {
^
/dataset/bad_randomness/lottery.sol:Lottery:42:10
bets.push(Bet(msg.value, block.number, won))
^
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 253.270756006 seconds ---
INFO:symExec:	====== Analysis Completed ======

  .oooooo.             o8o            o8o          
 d8P'  `Y8b            `"'            `"'          
888      888  .oooo.o oooo  oooo d8b oooo   .oooo.o
888      888 d88(  "8 `888  `888""8P `888  d88(  "8
888      888 `"Y88b.   888   888      888  `"Y88b. 
`88b    d88' o.  )88b  888   888      888  o.  )88b
 `Y8bood8P'  8""888P' o888o d888b    o888o 8""888P'


!!! SYMBOLIC EXECUTION TIMEOUT !!!
timeout